language-server/modules/langserver-core/src/test/resources/formatting/workerInteractions.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/io;
```

Comment:
Copyright header missing.

Reply:
these are test files for formatting @hemika ... these files are used for java level testing for formatting functionality ...  is header necessary there?